import "ydb/core/protos/config.proto";

package NKikimrConfig;
option java_package = "ru.yandex.kikimr.proto";

message TServerlessProxyConfig {
    optional THttpProxyConfig HttpConfig = 1;

    optional TGRpcConfig GRpcConfig = 2;
    optional string CaCert = 3;

    optional string DatabaseServiceEndpoint = 4;
    optional uint32 DatabaseListRefreshPeriodSeconds = 5 [default = 10];

    optional bool TestMode = 6;

    optional TLogConfig LogConfig = 7;
}

